src/web/Text.tsx

```diff
@@ -180,6 +180,10 @@ export class Text extends TextBase {
             }
         }
     }
+
+    getSelectedText(): string {
+        return ''; // Needs implementation (requires support from RN).
```

Comment:
I presume this can be implemented on the web. In any event, the comment is misleading.